PLAYER/SetPlayerMayOnlyEnterThisVehicle.md

```diff
@@ -8,8 +8,9 @@ ns: PLAYER
 void SET_PLAYER_MAY_ONLY_ENTER_THIS_VEHICLE(Player player, Vehicle vehicle);
 ```
 
+Limit the player to only enter this vehicle. Note set vehicle to false if you want them to access any vehicle.
 
 ## Parameters
-* **player**: 
-* **vehicle**: 
+* **player**: The player index, or -1 to get the local player ped.
```

Comment:
Usage of `-1` is discouraged, usually `PlayerPedId()` is used in Lua in favor of it.